packages/collector/test/tracing/frameworks/express/test.js

```diff
@@ -17,129 +17,132 @@ const globalAgent = require('../../../globalAgent');
 
 const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
 
-mochaSuiteFn('tracing/express', function () {
-  this.timeout(config.getTestTimeout());
+['latest', 'v5-beta'].forEach(version => {
```

Comment:
question: Did you change anything in the test? Its hard to review because of the indentions :)

Reply:
I haven't made any changes to the test except for including the `v5-beta` here

Reply:
Thank you!